src/main/java/org/opentripplanner/model/plan/WalkStep.java

```diff
@@ -34,7 +34,7 @@ public final class WalkStep {
   private final WgsCoordinate startLocation;
   private final double distance;
   private final RelativeDirection relativeDirection;
-  private final I18NString streetName;
+  private final I18NString name;
```

Comment:
A WalkStep does not have a name, it is the walkStep street that has a name so the prefix is should be there.

Reply:
The thing is that the walk step doesn't only refer to streets anymore.

In a previous PR I made it refer to station entrances and signs.  New instructions can include:

- "Enter station at Oslo Sentral, entrance 3"
- "Follow signs to platform 10"

Let's take a few minutes in the dev meeting to talk about what this property should be called.

Reply:
We will use something like "directionText"